lib/CompilerImpl/ClangLibCompiler.cpp

```diff
@@ -472,8 +504,12 @@ ClangLibCompiler::runOptimizer(const std::filesystem::path &src_IR,
   } 
 #endif
   else {
+#if LLVM_VERSION_MAJOR < 16
```

Comment:
Did you mean 18 or was 16 intentional?

Reply:
my mistake, should be 18

Reply:
I still see 16 here, unresolving till it is fixed.

Reply:
Fixed in 2368c43